pkg/util/utils.go

```diff
@@ -1184,7 +1184,7 @@ func ExitCode(err error) int {
 }
 
 func GetIdentityPath(name string) string {
-	return filepath.Join(homedir.Get(), ".ssh", name)
+	return filepath.Join(homedir.Get(), ".local", "share", "containers", "podman", "machine", name)
```

Comment:
I should have said earlier, we can actually use the shared functions to access the datadir...will do a patch.

Reply:
https://github.com/containers/podman/pull/21172